doc-src/dev.rst

```diff
@@ -555,9 +559,51 @@ Writing English
 * Use spell check. Keep your personal dictionary updated so your editor is not
   filled with false positives.
 
+.. _dependency-policy:
+
+Dependency policy
+-----------------
+
+Specific dependencies (prerequisites) are stated elsewhere in the
+documentation. This section describes our policy on which dependencies are
+acceptable.
+
+Generally
+~~~~~~~~~
+
+All dependencies must be stated and justified in the documentation.
+
+We want Charliecloud to run on as many systems as practical, so we work hard
+to keep dependencies minimal. However, because Charliecloud depends on new-ish
+kernel features, we do depend on standards of similar vintage.
+
+Core functionality should be available even on small systems with basic Linux
+distributions, so dependencies for run-time and build-time are only the bare
+essentials. Exceptions, to be used judiciously:
+
```

Comment:
We position ourselves to appeal to HPC environments, should our dependencies reflect the lowest common denominator (e.g., posix shell)? Have we looked at our competitor's dependencies?

Reply:
Not really but I assumed they're way bigger.